matRadGUI.m

```diff
@@ -115,6 +115,16 @@ function matRadGUI_OpeningFcn(hObject, ~, handles, varargin)
 axis equal off;
 set(f, 'AlphaData', alpha);
 
+% turn off the datacursormode (since it does not allow to set scrolling
+% callbacks
+handles.dcm_obj = datacursormode(handles.figure1);
+set(handles.dcm_obj,'DisplayStyle','window');
+if strcmpi(handles.dcm_obj.Enable,'on')
```

Comment:
Don't know if this line of code will be compatible with Matlab <2014... Usually you need to work with set and get there if you have three levels within a handles object-> strcmpi(get(handles.dcm_obj,'Enable'),'on').... Or is this a custom struct and not a property?